development/src/components/codingEditor.js

```diff
@@ -8,15 +7,14 @@ function CodingEditor() {
   const { theme, setTheme } = useTheme();
   const { items } = useTabContext();
   const language = items.filter((e) => e.active)[0].ext;
-  const [code, setCode] = useState(
-    '// Welcome to P2P Coder, a Community Prepared Platform...'
-  );
 
   const editorDidMount = (editor, monaco) => {
     editor.focus();
   };
 
-  const onChange = (newValue, e) => {};
+  const onChange = (newValue, e) => {
+    items.filter((e) => e.active)[0].code = newValue;
```

Comment:
This stores code state in `localStorage` item object `code` onChange to allow code state persistence. Great! 

This same function can simultaneously keep code state in a DB `document` specific to the collaboration session too at some point. I guess @Remi-dee is working on that!!

LG!